Analysis/Tasks/PWGHF/HFTrackIndexSkimsCreator.cxx

```diff
@@ -74,7 +74,7 @@ struct SelectTracks {
       int status_prong = 3; // selection flag , 2 bits on
 
       auto trackPt = track.pt();
-      if (b_dovalplots) {
+      if (b_dovalplots.value) {
```

Comment:
I would like to figure out whether this is necessary. @aalkin could you please comment whether this a feature or a bug?

Reply:
It should implicitly convert to underlying type in the right context, so the compiler must consider this ambiguous somehow. Better way would be to use explicit cast with: `(bool)b_dovalplots` or `b_dovalplots == true`.